demos/JWST/CalibratedStellarSpectra/S2_nirspec_fs_template.ecf

```diff
@@ -0,0 +1,21 @@
+# Eureka! Control File for Stage 2: Data Reduction
```

Comment:
Do we want to have calibrated spectra demo files for all the other instruments as well?

Reply:
This is a good long-term goal.  I'll see what I have on hand, but it won't be complete.